25.10/MyCoolProj2/Deserialize.cs

```diff
@@ -0,0 +1,32 @@
+﻿#nullable enable
+using System.IO;
+using System.Text.Json;
+using Microsoft.Extensions.Configuration;
+using MyCoolProj2.Pokemons;
+using Newtonsoft.Json;
+
+namespace MyCoolProj2
+{
+    public class Deserialize
```

Comment:
Use subjects for naming classes, interfaces, etc. and verb for methods
So this class is better to call Deserializer and actually since it deserializes only Pokemon model even better would be PokemonDeserializer

Also, create interface and inject this as dependency using DI to the places of useage